inc/block-editor.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Block editor (gutenberg) specific functionality
+ *
+ * @package UnderStrap
+ */
+
+
+add_action( 'after_setup_theme', 'understrap_block_editor_setup' );
+
+if ( ! function_exists( 'understrap_block_editor_setup' ) ) {
+
+	/**
+	 * Sets up our default theme support for the WordPress block editor.
+	 *
+	 */
+	function understrap_block_editor_setup() {
+
+		// Add support for the block editor stylesheet.
+		add_theme_support( 'editor-styles' );
+
+		// Add support for wide alignment.
+		add_theme_support( 'align-wide' );
+
+		// Register our custom colors as options in the editor.
+		$color_palette = understrap_generate_color_palette();
+		if ( $color_palette ) {
+			add_theme_support( 'editor-color-palette', $color_palette );
+		}
+	}
+}
+
+if ( ! function_exists( 'understrap_generate_color_palette' ) ) {
+	/**
+	 * Checks for our JSON file of color values. If exists, creates a color palette array.
+	 *
+	 * @return array|bool
+	 */
+	function understrap_generate_color_palette() {
+		$color_palette = array();
+
+		// Grabs the autogenerated color palette that we're pulling from our compiled bootstrap stylesheets.
+		$color_palette_json = wp_remote_get( get_stylesheet_directory_uri() . '/inc/editor-color-palette.json' );
```

Comment:
```suggestion
		$color_palette_json = wp_remote_get( get_theme_file_uri() . '/inc/editor-color-palette.json' );
```

A child theme not shipping its own /inc/editor-color-palette.json will throw an error when using `get_stylesheet_directory_uri()`.

Reply:
This is a good suggestion.